yellowbrick/features/rankd.py

```diff
@@ -126,18 +172,13 @@ class Rank2D(FeatureVisualizer):
     process, but can and should be set as early as possible.
     """
 
-    ranking_methods = {
-        'pearson': lambda X: np.corrcoef(X.transpose()),
-        'covariance': lambda X: np.cov(X.transpose()),
-    }
-
-    def __init__(self, ax=None, algorithm='pearson', features=None,
+    def __init__(self, ax=None, algorithm='shapiro', features=None,
```

Comment:
The default here should probably be `algorithm=None` since both `shapiro` and `pearson` are subclass-specific.